datamodel/include/datamodel/math/Vector.hpp

```diff
@@ -216,6 +238,13 @@ static void from_json(const nlohmann::json &j, rcbe::math::Vector2d &v);
 };
 }
 
+namespace rcbe::binary {
+template <>
```

Comment:
Try to remove specializations

Reply:
Not gonna work either way, there has to be a specializtion. Either of a struct (like in nlohmann) or like this